### ISSUE TITLE
Load different csv files from command line without prefix

### ISSUE THREAD

**Issue by RobinMaub** [facontidavide/PlotJuggler#1000]
Hi everyone,

I am using the 3.9.0 version on Windows. I would like load several csv files in a single command line, but I do not want to add a prefix and stay stucked on the "Add prefix" page, is it possible ?

Thanks in advance

**Comment by matthewgcooper** [comment]
Yes, this would be great. I have the opposite issue where I try to open multiple data files from the command line like:

`plotjuggler -d file1.csv -d file2.csv`

and I get no prefix option. It would be good if there was a command line option to enable/disable the prefix option.
